088-rebalance-progress-status

```diff
@@ -0,0 +1,328 @@
+# Partition Rebalance Progress Status
+
+This proposal introduces a new feature to monitor the progression of an ongoing partition rebalance executed by a Strimzi-managed Cruise Control instance via a `KafkaRebalance` custom resource. 
+
+## Current Situation
+
+At this time, Strimzi users are able to execute partition rebalances via `KafkaRebalance` custom resources but can only monitor the progression of those partition rebalances in two ways outside the `KafkaRebalance` resource by:
+
+- Manually querying the Cruise Control REST API endpoint directly. 
+- Inspecting the logs of the Cruise Control instance. 
+
+Unfortunately, neither of these methods are particularly user friendly. 
+
+## Motivation
+
+Information surrounding the progress of an executing partition rebalance is useful for planning future cluster operations. Knowing things like:
+
+- How much time an ongoing partition rebalance has left to take
+- How much data an ongoing partition rebalance has left to transfer 
+
+helps users understand the cost of an ongoing partition rebalance, decide whether or not they should continue or cancel it, and know when future operations will be able to be safely executed.
```

Comment:
Style Nit: I am not sure the bullet points make things clearer? Feels a bit disjointed when you read it. Maybe just make this a single sentence?